docs/apis-clients/community-clients/rust.md

```diff
@@ -4,7 +4,9 @@ title: "Rust"
 description: "Take a deeper look at the source code, Rust crate, and a podcast interview alongside Rust."
 ---
 
+:::note
 The Rust client, Zeebest, was previously maintained by [Mackenzie Clark](https://github.com/xmclark), and is currently seeking a new maintainer!
+:::
 
 - [Source code](https://github.com/camunda-community-hub/zeebest)
```

Comment:
@npepinpe @Zelldon -- I'm a little confused here. Is [this repo](https://github.com/camunda-community-hub/zeebe-client-rust) better suited as the default for source code, or just as an added tab?

Reply:
Hmm good question. Zeebest ist AFAIK out dated / unmaintained. The other is newer, not sure whether @pihme or Ole will continue to work on it.

Reply:
I plan to continue working on it, but it will probably be a slow burner. At the moment, I don't think https://github.com/camunda-community-hub/zeebe-client-rust is mature enough to be mentioned here.

Reply:
@pihme Would you remove this page entirely then?